### ISSUE TITLE
Crash in iOS 8 

### ISSUE THREAD

**Issue by alexiss03** [twilio/video-quickstart-objc#42]
Hi! We've implemented Twilio Video in our existing app. It works okay with iOS 9, 10. However, when we run it in an iOS 8 device, the memory allocation via Instruments showed that Twilio Video keeps allocating memory until the app produces the memory warning and eventually crashes.

**Comment by ptankTwilio** [comment]
@alexiss03 Thanks for reaching out to us. Sorry for the crash.

The crash is caused by buffer leaks in iOS 8.x devices while rendering the local video. We have the fix and it is being validated by our QA team currently. We should be able to release fix in a few days. I will let you know when we release it.  In the mean time you can try following workaround on iOS 8.x devices - 

At [ViewController.mm line#133]( https://github.com/twilio/video-quickstart-objc/blob/master/ObjCVideoQuickstart/ViewController.m#L133), replace 
```
        [self.localVideoTrack attach:self.previewView];
```
by following
```
        TVIVideoViewRenderer *renderer =  [TVIVideoViewRenderer rendererWithRenderingType:TVIVideoRenderingTypeOpenGLES delegate:nil];
        [self.localVideoTrack addRenderer:renderer];
        renderer.view.frame = self.previewView.bounds;
        [self.previewView addSubview:renderer.view];
```
Let me know if you have any questions.

**Comment by alexiss03** [comment]
Hi @ptankTwilio. I'm still getting the crash due to memory using the workaround you suggested. Can you give me an estimate of the release of your next build? Thanks a lot.

**Comment by ptankTwilio** [comment]
@alexiss03  We have just released Video SDK version 1.0.0-beta8 SDK which has a fix for iOS 8.0 memory leak issue. 

Here are the 1.0.0-beta8 change logs  : https://www.twilio.com/docs/api/video/changelogs/ios#100-beta8-february-17-2017

Please let me know if you have any questions.

**Comment by alexiss03** [comment]
@ptankTwilio The memory works okay now.  Thanks for the quick response fix!
